<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ORDENACAO</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="0" colspan="4">
     <widget class="QTextEdit" name="textEdit_Out">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="undoRedoEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_Put"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_Count"/>
    </item>
    <item row="1" column="2" colspan="2">
     <widget class="QPushButton" name="pushButton_Sort">
      <property name="text">
       <string>Sort</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_Count">
      <property name="text">
       <string>QUANTIDADE QUE  IR√Å SER ORDENADA --&gt; </string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_Put">
      <property name="text">
       <string>INSERIR --&gt;</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2" colspan="2">
     <widget class="QPushButton" name="pushButton_Random">
      <property name="text">
       <string>Aleatorio</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="3">
     <widget class="QComboBox" name="comboBox_Options">
      <item>
       <property name="text">
        <string>Bubble</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Heap</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Insert</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Merge</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Quick</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Selection</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
